upstart/slack.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=service handling slack connection
+Wants=network-online.target
+After=network.target network-online.target NetworkManager.service NetworkManager-wait-online.service
+
+[Service]
+Type=simple
+RemainAfterExit=no
+ExecStart=/bin/bash -c "/usr/bin/tmux -2 new-session -d -s slack && \
+    /usr/bin/tmux -2 send -t slack 'python `rospack find slack`/scripts/ros_service.py'
```

Comment:
this script needs [`sys.args`](https://github.com/unity-robotics/msh/pull/259/files#diff-d119f6fdebb87289a5673dc186d04aadR195) which represent [`token`, `channel`, `username`](https://github.com/unity-robotics/msh/pull/259/files#diff-d119f6fdebb87289a5673dc186d04aadR29)

those arguments are robot specific and need to be adjusted before this service can be installed!